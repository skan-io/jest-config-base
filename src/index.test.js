import config from '.';

test('jest-config regression', ()=> {
  expect(config).toMatchSnapshot();
});
