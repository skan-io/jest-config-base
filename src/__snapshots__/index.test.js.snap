// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-config regression 1`] = `
Object {
  "clearMocks": true,
  "collectCoverage": true,
  "collectCoverageFrom": Array [
    "src/**/*.js",
  ],
  "coverageDirectory": "./build/cov",
  "coverageReporters": Array [
    "lcov",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 100,
      "functions": 100,
      "lines": 100,
      "statements": 100,
    },
  },
  "modulePathIgnorePatterns": Array [
    "<rootDir>/build/",
  ],
  "resetMocks": false,
  "setupFiles": Array [],
  "testMatch": Array [
    "<rootDir>/src/**/*.test.js",
  ],
  "testPathIgnorePatterns": Array [
    "<rootDir>/node_modules/",
    "<rootDir>/build/",
  ],
  "timers": "fake",
  "transform": Object {
    "^.+\\\\.js$": "babel-jest",
  },
  "transformIgnorePatterns": Array [
    "<rootDir>/node_modules/",
    "<rootDir>/build/",
  ],
  "watchPathIgnorePatterns": Array [
    "<rootDir>/node_modules/",
    "<rootDir>/build/",
  ],
}
`;
